build-tools-internal/src/main/groovy/elasticsearch.build-complete.gradle

```diff
@@ -9,8 +9,8 @@
 import java.nio.file.Files
 
 String buildNumber = System.getenv('BUILD_NUMBER')
-
-if (buildNumber) {
+String performanceTest = System.getenv('BUILD_PERFORMANCE_TEST')
+if (buildNumber && performanceTest != null) {
```

Comment:
Do we know whether anyone is currently using these uploaded files e.g. to investigate ES issues? In which case, would they now need to run the build benchmarks to get these files?

Reply:
Yeah, I think this condition is simply inverted. We want to generate the file when we are **not** running a performance test. So we should trigger when `performanceTest == null`.

Reply:
fixed